### PR TITLE
[8.0][IMP][crm] Add hook for preparing scheduled phonecall values and allow inheritance

### PR DESCRIPTION
**Pull request by antespi**
This PR does not change Odoo behavior but adds a hook in order to inheritance it in other addons
## 

I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/crm/crm_phonecall.py [hunk 1]
```diff
@@ -110,6 +110,23 @@ def compute_duration(self, cr, uid, ids, context=None):
                 self.write(cr, uid, [phonecall.id], values, context=context)
         return True
 
+    def _prepare_another_phonecall(self, cr, uid, call, schedule_time, call_summary, \
```
Comment (pedrobaeza) [inline addons/crm/crm_phonecall.py]: Put at the end of the name `_vals` for following convention

**Comment by antespi** [comment]
@pedrobaeza thanks for your review. Changes done

**Comment by pedrobaeza** [comment]
As the method returns multiple records, it's difficult to match each one with the proper source, so this hook can be very convenient, so :+1:

**Comment by rafaelbn** [comment]
:+1:

**Comment by tivisse** [comment]
Hello,

Thank you for your contribution but the branch is no longer supported. We only support the last 3 stable versions so no longer accepts patches into this branch.
We apology if we could not look at your request in time.
If the contribution still makes sense for the upper version, please let us know and do not hesitate to recreate one for the recent versions. We will try to check it as soon as possible.

Regards

**Pull request closed without merging**